packages/hs-react-ui/README.md

```diff
@@ -63,3 +238,27 @@ const MyContainer = styled(Div)`styles here`;
 ### Enabling Theming
 
 Theming is enabled through props of each component. When building a new component, or adding a styled-component to an existing component, ensure that there is a matching prop to accept an override, and that styled-component is a named export from the component. See the [Card component](https://github.com/Headstorm/rasa-ui/tree/master/packages/hs-react-ui/src/components/Card) as an example.
+
+## State of CICD workflow
+###What's working?
+- Pushes to master trigger a release build
+- Auto versioning
+- Pushes to v1-release trigger a pre-release build
+build checks on every PR to master and v1-release
+- Change logs are generated (see link below)
+- storybook docs are generated and checked in for releases
+- Test coverage
+
+###What isn't working yet?
+- Publishing to NPM isn't working yet due to the authentication token not having proper permissions - we have a ticket submitted to get this fixed and Bennett has been doing an excellent job tackling those types of tickets for us :clapping:
```

Comment:
This part is working now and can be removed 😄

Reply:
Nice. Update. Thanks, DJ